app/views/data_tables/_data_table.html.haml

```diff
@@ -243,17 +244,6 @@
           });
         }
       },
-      height: function() {
```

Comment:
Are you sure the heigh t function is not needed anymore? I know RC tinkered with this a while back to get some stuff working, maybe displaying 2 grids on a screen or something.

Reply:
I went through this yesterday. It's needed. . The height warning only affects 22.2.

Reply:
From my testing last night, it seems the height function is not needed. The height of the grid can be set using the resize function. It would be better if we could use pure css for this, but without changing to a flex layout, I don't see how we can do that(Might not have thought through every possibility yet though).

The reason the resize function works now without the height function, is because of adding the window `load` listener. It sets the height on window load, and then on any resize, it will change the height.

Ps. The attempt that Ife did previously was not correct, and would not have fixed the issue.

Reply:
Can you test the workflow controls second grid when it's got more than 200 records. I had an issue where the second grid kept on growing and the scroll bar didn't work correctly.